src/index.html

```diff
@@ -22,6 +22,377 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <!-- <h1>Catalog</h1> -->
+    <header class="header">
+      <img
+        src="images/logo.png"
+        alt="Moyo logo image"
+        class="header__logo"
+      />
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#section-1"
+            >
+              APPLE
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#section-2"
+            >
+              SAMSUNG
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#section-3"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#section-4"
+              data-qa="nav-hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#section-5"
+            >
+              GADGETS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#section-6"
+            >
+              TABLETS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#section-7"
+            >
+              PHOTO
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#section-8"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="container">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            src="./images/imac.jpeg"
```

Comment:
The image source './images/imac.jpeg' is repeated across multiple cards. If this is intentional, ensure that each card is meant to display the same image. Otherwise, consider using different images for each product card.